packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -23,23 +24,10 @@ const BIContractActivationWindow = ({
   onAccountDeleted
 }) => {
   const [extraParams, setExtraParams] = useState(null)
-  const [isWindowVisible, setWindowVisible] = useState(false)
-  const [isDeleteConnectionDialogVisible, setDeleteConnectionDialogVisible] =
-    useState(false)
-  const [shouldRefreshContracts, setShouldRefreshContracts] = useState(false)
-
   const konnectorPolicy = findKonnectorPolicy(konnector)
   const client = useClient()
 
-  const onClose = useCallback(async () => {
-    client.dispatch({
-      type: 'RECEIVE_MUTATION_RESULT',
-      mutationId: client.generateRandomId(),
-      response: { data: { ...account, _deleted: true } },
-      definition: Mutations.deleteDocument({ ...account, _deleted: true })
-    })
-    onAccountDeleted()
-  }, [onAccountDeleted, account, client])
```

Comment:
nit: you should have reverted the commit that added this piece of code before doing anything else.